src/ray/util/container_util.h

```diff
@@ -93,10 +103,19 @@ std::ostream &operator<<(std::ostream &os, DebugStringWrapper<std::tuple<Ts...>>
   return os;
 }
 
+template <typename T, std::size_t N>
+std::ostream &operator<<(std::ostream &os, DebugStringWrapper<std::array<T, N>> c) {
```

Comment:
do we have any usage of this specialization?

Reply:
In this PR, I'm trying to add well-known types to debug string.